Source/WebCore/page/EventHandler.h

```diff
@@ -462,6 +462,8 @@ class EventHandler final : public CanMakeCheckedPtr<EventHandler> {
     enum class Cancelable : bool { No, Yes };
     bool dispatchMouseEvent(const AtomString& eventType, Node* target, int clickCount, const PlatformMouseEvent&, FireMouseOverOut);
 
+    bool swallowClickOrAuxclickEvent(const PlatformMouseEvent&, const MouseEventWithHitTestResults&);
```

Comment:
Maybe `swallowAnyClickEvent` or `swallowEitherClickEvent`? Names without "and" seems to read more smoothly, even if they are a tad less precise.